### PR TITLE
Security specification

### PR DESCRIPTION
**Pull request by MatShir**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | specs
| Fixed ticket? | [Fixes #{issue number here} if there is a related issue](https://github.com/PrestaShop/PrestaShop/pull/21120)

### CODE REVIEW

File: content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+There are 10 columns:
+
+- ID of the session
+- Customer ID
+- Customer's First name
+- Customer's Last name
+- Customer's Email
+- Last update
+- Actions: Delete the session
```
Comment (MatShir) [inline content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md:53]: @PierreRambaud Does it delete the session even if it is not outdated ?

File: content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+## Bulk actions drop-down
+
+This dropdown is enabled only if at least one session is selected, otherwise, it's disabled.
+
+There is 1 option:
+- Delete selection: it deletes the session. The customer can't log back with the session deleted. 
+
+### Select all checkbox
```
Comment (PierreRambaud) [inline content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md:37]:
```suggestion
### Select all checkboxes
```

File: content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md (hunk 1 above)
Comment (PierreRambaud) [reply content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md:53]: Yes, it forces the deletion in database and the disconnect

File: content/1.7/back-office/advanced-parameters/Security/employee-sessions-listing.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+## Bulk actions drop-down
+
+This dropdown is enabled only if at least one session is selected, otherwise, it's disabled.
+
+There is 1 option:
+- Delete selection: it deletes the session. The employee can't log back with the session deleted. 
+
+### Select all checkbox
```
Comment (PierreRambaud) [inline content/1.7/back-office/advanced-parameters/Security/employee-sessions-listing.md:37]:
```suggestion
### Select all checkboxes
```

File: content/1.7/back-office/advanced-parameters/Security/security.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Back office token protection - toggle - By default is enabled. Enbaled, the navagation in the back office can be done with a valid token.
```
Comment (PierreRambaud) [inline content/1.7/back-office/advanced-parameters/Security/security.md:1]:
```suggestion
Back office token protection - toggle - By default is enabled. Enabled, a token is added in the URL during the navigation in the back office, if the token is invalid, it displays a warning with a message asking if the user really wants to display this page.
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by MatShir** [comment]
> @PierreRambaud
> If the employee use a sessions on computer A and the computer B,
> And the user deletes the employee session of the computer A.
> Can the employee log with the session of the computer B or he has to log in again completely

UP

**Comment by MatShir** [comment]
thanks for the reply

**Comment by MatShir** [comment]
Sorry my comment was pending 😓

File: content/1.7/back-office/advanced-parameters/Security/employee-sessions-listing.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+## Bulk actions drop-down
+
+This dropdown is enabled only if at least one session is selected, otherwise, it's disabled.
+
+There is 1 option:
+- Delete selection: it deletes the sessions after validating the validation's modal. The employee can't log back with the session deleted. 
+
+### Select all checkboxes
+
+After checking the box, all the products displayed on the page are selected or unselected.
```
Comment (eternoendless) [inline content/1.7/back-office/advanced-parameters/Security/employee-sessions-listing.md:39]:
```suggestion
After checking the box, all the rows displayed on the page are selected or unselected.
```

**Pull request merged**